src/js/select2/i18n/fa.js

```diff
@@ -0,0 +1,38 @@
+/* jshint -W100 */
```

Comment:
Where was W100 being triggered? I don't see why this error would be triggered in this file, considering it hasn't been triggered in other files.